controllers/recruitersController.js

```diff
@@ -0,0 +1,15 @@
+const recruiterQuery = require('../models/recruiters')
+
+const addRecruiter = async (req, res) => {
+  try {
+    const result = await recruiterQuery.addRecruiterInfo(req)
```

Comment:
We should pass req.body.
Also, we should validate the data.

Reply:
We need to access timestamp so 'req' has been passed over there.

Reply:
I have used req.body and req.params.username instead of req.